_posts/2020-02-14-running-a-great-socratic-seminar.md

```diff
@@ -0,0 +1,70 @@
+- Critical vulnerabilities in popular hardware and software
+- Network statistics and phenomena (e.g. oscillations in the mempool/difficulty/UTXO set, reorgs, 51% attacks, double spends)
+- Mailing list posts with interesting proposals or critiques
+- Pull requests from popular repositories that show substantial changes in behavior or architecture as well as improve security or performance.
+- Research papers with substantial insights into topics relevant to bitcoiners
+- Blog posts that investigate topics of privacy, software development, cryptography and more. 
+
+Thankfully there are many socratic style meetups worldwide which post their content for public consumption; going through their recent events will give you a lots of materials to choose from [LINK][LINK[LINK]. Once you've collated the material you would like to cover and are preparing it for discussion, we recommend highlighting important content. Highlights are a good mnemonic device for the moderator and they help drive discussion from the crowd. Popular browser platforms have plugins that allow you to highlight copy on the page. 
+
+Be careful not to use the highlights as a crutch--if you exclusively read word-for-word from the links and do not engage the crowd when presenting the material, you will lose their attention and discussion may fall flat. In general we do not recommend regurgitating the content into a powerpoint--people want to see the source material and you will want to avoid bloating your preparation time generating slides. 
+
+In terms of ordering the content, we recommend you arrange the tabs in your browser from left to right, starting with the things you would like to cover first. Avoid starting the event with very esoteric/detailed subject matter. Find something that has relevance for the whole group and is ripe for discussion. This will warm up the room and get everyone engaged. From here you can dig more deeply into highly technical protocol-specific subject matter (e.g. pull requests, BIPs, BOLTs, etc..). Once through some of the content with a high degree of technicality, you can soften the end of the meetup by covering/demoing fun projects (e.g. lightning games). We recommend using a browser plugin to save your tabs in the event your browser or browser crashes, device loses power etc.
+
+With links selected, tabs ordered and content highlighted, it is time to prepare yourself to lead discussion. Reading the material is the first step. Once you have read and highlighted the material, you should practice explaining it--this will uncover holes in your knowledge that were not previously apparent. The more you prep, the more enjoyable the event will be, both for you and the attendees. Ultimately, your goal is to have just enough of an understanding of the material so that you can contextualize the content in a way which avoids the attendees feeling completely lost. 
```

Comment:
yourself to lead discussion -> yourself to lead the discussion

Reply:
```suggestion
With links selected, tabs ordered and content highlighted, it is time to prepare yourself to lead the discussion. Reading the material is the first step. Once you have read and highlighted the material, you should practice explaining it--this will uncover holes in your knowledge that were not previously apparent. The more you prep, the more enjoyable the event will be, both for you and the attendees. Ultimately, your goal is to have just enough of an understanding of the material so that you can contextualize the content in a way which avoids the attendees feeling completely lost. 
```